### PR TITLE
CI fixes

### PR DESCRIPTION
**Pull request by lmars**

### CODE REVIEW

File: test/cluster/cluster.go [hunk 1]
```diff
@@ -500,6 +502,19 @@ func lookupUser(name string) (int, int, error) {
...
 
 func (c *Cluster) DumpLogs(w io.Writer) {
+	done := make(chan struct{})
+	go func() {
+		c.dumpLogs(w)
+		close(done)
+	}()
+	select {
+	case <-done:
+	case <-time.After(60 * time.Second):
```
Comment (heavenlyhash) [inline test/cluster/cluster.go]: This timeout covers all the log gather attempts at once... do we need the big one, if the timeout per ssh connection is now low?

Comment (titanous) [inline test/cluster/cluster.go]: The ssh timeout only covers the connect time, this covers execution time too.

Comment (heavenlyhash) [inline test/cluster/cluster.go]: OH.... so if the dialSSH call times out or errors once, it doesn't stop future runs from trying and timing out again.  Maybe that's not the greatest behavior, maybe we should just set an "error and stop trying" flag for that?  WDYT?

Comment (titanous) [inline test/cluster/cluster.go]: I think this is fine for now.

**Comment by heavenlyhash** [comment]
LGTM

**Comment by titanous** [comment]
LGTM

**Comment by archSeer** [comment]
LGTM

**Pull request merged**